README.md

```diff
@@ -1,6 +1,10 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+For instructions, 
+--you need to install dotenv package.
+-- make two  files starts with ".env", one for development perposes one for testing . 
+--then set the environment variables  with your database name considering the testing and developing database names . 
+--make sure they are ignored from git. 
```

Comment:
You have already done this for them 😄 Your .gitignore is in the root of this directory.